feat(icon): add opentofu icon

kieran-lowe opened the pull request:
<!-- markdownlint-disable MD041-->

<!-- Please first read how to submit a pull request, if you haven't already done so.
https://github.com/vscode-icons/vscode-icons/wiki/PullRequest -->

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

PR to add the OpenTofu (Open-source fork of HashiCorp Terraform) icon for `.tofu` etc.

- Homepage: https://opentofu.org/
- GitHub Repo: https://github.com/opentofu/opentofu
- File extensions: https://opentofu.org/docs/language/files/
- Test file extensions: https://opentofu.org/docs/cli/commands/test/
- Icon used from here: https://github.com/opentofu/brand-artifacts/tree/main/symbol-only/transparent/SVG
  - Used the guide [here](https://github.com/vscode-icons/vscode-icons/wiki/Contributing) to format it to required conventions

I think I've done it correctly, first time using Inkscape so let me know if otherwise!

kieran-lowe commented:
Rebased PR

remcohaszing commented on icons/file_type_opentofu.svg:
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32"><path fill="#0d1a2a" fill-rule="evenodd" d="M17.06.61a2.21 2.21 0 0 0-2.12 0L4.1 6.57l-.03.01-1.42.79A2.18 2.18 0 0 0 1.5 9.29v13.4c0 .81.44 1.55 1.14 1.94l10.95 6.02.05.02 1.3.72c.66.36 1.46.36 2.12 0l1.31-.72.04-.02 10.95-6.02a2.2 2.2 0 0 0 1.14-1.92V9.29c0-.8-.44-1.54-1.14-1.92l-1.42-.78-.03-.01zm.7 15.39 10.14-5.58.03-.01.13-.07a. ...
```
One suggestion to make it a bit smaller:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path fill="#0d1a2a" d="M17.06.61a2.21 2.21 0 0 0-2.12 0L4.1 6.57l-.03.01-1.42.79A2.18 2.18 0 0 0 1.5 9.29v13.4c0 .81.44 1.55 1.14 1.94l10.95 6.02.05.02 1.3.72c.66.36 1.46.36 2.12 0l1.31-.72.04-.02 10.95-6.02a2.2 2.2 0 0 0 1.14-1.92V9.29c0-.8-.44-1.54-1.14-1.92l-1.42-.78-.03-.01zm.7 15.39 10.14-5.58.03-.01.13-.07a.5.5 0 0 1 .74.44v10.44a.5.5 0 0 1-.74.44l-.1-.05-.06-.03zm-13.7-5.6.03.02 10.15 5.57L4.1 21.58l-.06.03-.1.05a.5.5 0 0 1-.74-.44V10.78a.5.5 0 0 1 .74-.45l.13.07zM27.1 8.06l-9.5-5.23a.5.5 0 0 0-.76.43v11.31L27.1 8.94a.5.5 0 0 0 0-.88zm-22.2.87 10.25 5.63V3.26a.5.5 0 0 0-.75-.43L4.9 8.06a.5.5 0 0 0 0 .87zm0 15.01a.5.5 0 0 1-.02-.87l10.28-5.64v11.3c-.01.38-.4.61-.74.45l-9.51-5.23zm11.94 4.78v-11.3l10.28 5.64a.5.5 0 0 1-.03.88l-9.51 5.23a.5.5 0 0 1-.74-.44z"/><path fill="#e6c220" d="M15.75 2.08a.5.5 0 0 1 .5 0l10.84 5.97c.35.19.35.7 0 .89L16.24 14.9a.5.5 0 0 1-.49 0L4.91 8.94a.5.5 0 0 1 0-.9z"/><path fill="#feda15" d="M3.19 10.78a.5.5 0 0 1 .75-.45l10.95 6.02c.16.1.26.26.26.44v11.93a.5.5 0 0 1-.75.44L3.46 23.15a.51.51 0 0 1-.26-.44V10.78z"/><path fill="#fff" d="M28.06 10.33a.5.5 0 0 1 .74.45V22.7a.5.5 0 0 1-.26.44L17.6 29.16a.5.5 0 0 1-.75-.44V16.79c0-.18.1-.35.26-.44z"/><path fill="#0d1a2a" d="M7.79 20.13v.02L5.22 18.8v-.02c.06-.8.68-1.15 1.39-.77s1.24 1.32 1.18 2.12zM11.83 22.46v.02l-2.57-1.36v-.01c.06-.8.68-1.15 1.39-.77s1.24 1.32 1.18 2.12z"/></svg>
```

remcohaszing reviewed (changes requested):
Thanks! Looks great! I have one suggestion to make it even a bit smaller.

kieran-lowe commented:
> Thanks! Looks great! I have one suggestion to make it even a bit smaller.

Thanks @remcohaszing! Added your suggestion to make it smaller and rebased again.
